### ISSUE TITLE
null-ls breaks code actions prompt after 0.5.0 -> 0.5.1

### ISSUE THREAD

**Issue by CallumHoward** [jose-elias-alvarez/null-ls.nvim#229]
### FAQ

- [X] I have checked [FAQ](https://github.com/jose-elias-alvarez/null-ls.nvim#faq) and it didn't resolve my problem.

### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues?q=is%3Aissue) and there are no issues with the same problem.

### Neovim Version

NVIM v0.5.1

### Steps to reproduce

Possibly related: https://github.com/jose-elias-alvarez/null-ls.nvim/issues/205
Reddit thread: https://www.reddit.com/r/neovim/comments/pzj21j/vimlspbufcode_action_not_working_after_050_051/

After upgrading from 0.5.0 -> 0.5.1 when I call `lua vim.lsp.buf.code_action()` on a line that I am sure has a code action, nothing happens and there is no error.

Extra info
- I know a code action exists and the LSP is working because I have nvim-lightbulb.
- I know the code action can be applied because :TSLspFixCurrent from nvim-lsp-ts-utils which applies the code action.
- I'm not using lspsaga.nvim
- I can also use `:Telescope lsp_code_actions` to successfully apply the code action
- From Reddit thread it appears some others may be seeing the same issue
- The issue occurs even if null-ls is not attached to the current buffer

Config:
```lua
use("kabouzeid/nvim-lspinstall")
use("neovim/nvim-lspconfig")
use("jose-elias-alvarez/nvim-lsp-ts-utils")
use("jose-elias-alvarez/null-ls.nvim")

require("lspconfig")["null-ls"].setup({}) -- works if this is commented/removed

-- LspInstall setup
local function setup_servers()
  require("lspinstall").setup()
  local servers = require("lspinstall").installed_servers()
  for _, server in pairs(servers) do
    require("lspconfig")[server].setup({})
  end
end

setup_servers()
```
([full config](https://github.com/CallumHoward/dotfiles/blob/b8fff95c3e817eef9cf99826b99623c730514ce1/.config/nvim/lua/plugin_config.lua#L160))

Minimal code action repro in C and clangd language server (any other example should work too):
```c
int main() { return EXIT_SUCCESS; }
```

Expected: Calling `lua vim.lsp.buf.code_action()` on this line will have code action:
```
Add include <stdlib.h> for symbol EXIT_SUCCESS
```

### Expected behavior

I expect a list to be shown with the code actions when I call `lua vim.lsp.buf.code_action()`.

### Actual behavior

No error and no output or menu to select code action.

### Debug log

Log is empty after clearing and following the above repro

### Help

Yes, but I don't know how to start. I would need guidance

### Implementation help

Unsure how to diagnose the issue further. Perhaps it is due to configuration changes in LSP made between 0.5.0 and 0.5.1.

**Comment by jose-elias-alvarez** [comment]
Thanks for the thorough report. This is caused by the fact that 0.5.1 backported the new handler signature and should be fixed by 47776ebe18f9a56a8c15ce93992a61458c9cb8ef, but can you try updating the plugin and let me know if that takes care of it?

**Comment by CallumHoward** [comment]
Yep, seems like that fixes it. Thanks!
